src/Hintman.hs

```diff
@@ -1,6 +1,9 @@
 module Hintman
        ( someFunc
+       , LineNumber
        ) where
 
+type LineNumber = Natural
```

Comment:
Using `Natural` is pretty clever, but I'm not sure if it'll bring much advantage over `Int`, and `Int`
might end up being easier to work with...

Reply:
I agree that `Natural` is harder to work with, I was trying to have the types as close as possible to the domain. I'll let you guys decide how to resolve the trade-off :)

Reply:
Yeah, `Natural` is better choice but functions for list type work with `Int` unfortunately... So let's use `Int`